### PR TITLE
Changes for https support

### PR DESCRIPTION
**Pull request by Deadolus**
These are changes to support https

### CODE REVIEW

File: tools/multipleJobsToTschenggins.sh [hunk 1]
```diff
@@ -29,7 +29,13 @@ function update {
 	curl --header "Content-Type: application/json" -s \
 		--request POST \
 		--data '{"cmd":"update","debug":0,"states":[{"name":"'$3'","result":"'$1'","server":"'$TARGETSERVER'","state":"'$2'"}]}' \
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
+        -A "Tschenggins-Bash-Script" \
```
Comment (phkehl) [inline tools/multipleJobsToTschenggins.sh]: Maybe better -A "multipleJobsToTschenggins.sh/1.0" ?
To be in line with the other scripts and the typical agent string format.

File: tools/multipleJobsToTschenggins.sh [hunk 2]
```diff
@@ -29,7 +29,13 @@ function update {
 	curl --header "Content-Type: application/json" -s \
 		--request POST \
 		--data '{"cmd":"update","debug":0,"states":[{"name":"'$3'","result":"'$1'","server":"'$TARGETSERVER'","state":"'$2'"}]}' \
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
```
Comment (phkehl) [inline tools/multipleJobsToTschenggins.sh]: No leaks please! Remove any reference to our existing setup. Perhaps make it a command line option.

**Comment by Deadolus** [comment]
I addressed your comments. You were of course right. 
I force pushed a rewritten history to delete some information from github. 
If you are ok with the changes, you may now merge this updated branch.

**Comment by phkehl** [comment]
Merci!

**Pull request merged**